UnitTests/BraintreeVenmoTests/BTVenmoClient_Tests.swift

```diff
@@ -90,20 +91,9 @@ class BTVenmoClient_Tests: XCTestCase {
         waitForExpectations(timeout: 2, handler: nil)
     }
 
-    func testTokenizeVenmoAccount_whenReturnURLSchemeIsNil_logsCriticalMessageAndCallsBackWithError() {
+    func testTokenizeVenmoAccount_whenReturnURLSchemeIsNil_andCallsBackWithError() {
         let venmoClient = BTVenmoClient(apiClient: mockAPIClient)
         BTAppContextSwitcher.sharedInstance().returnURLScheme = ""
-
-        
-        var criticalMessageLogged = false
```

Comment:
Do we need to add new tests to replace these removed tests? Or is testing this logic handled somewhere else now

Reply:
Since the functionality of `BTLogger` is different now, I think it's ok to remove these tests. It may be worthwhile to create a new test file for "stupid" tests of `BTLogger` itself. @kaylagalway and I have started doing this for the other files!